app/styles/_data-toolbar.less

```diff
@@ -7,10 +7,31 @@
  .flex(@columns: 1 1 0%);
  .flex-wrap(@wrap: wrap);
  padding: 5px 0;
+ .data-toolbar-checkbox {
```

Comment:
have to override the margins to 0 or it increases the size of the toolbar, but otherwise yes, will switch to that instead since its fewer styles